reflex/components/layout/foreach.py

```diff
@@ -66,9 +71,9 @@ def render(self):
         tag = self._render()
         try:
             type_ = (
-                self.iterable._var_type
-                if self.iterable._var_type.mro()[0] == dict
-                else self.iterable._var_type.__args__[0]
+                tag.iterable._var_type
+                if tag.iterable._var_type.mro()[0] == dict
+                else tag.iterable._var_type.__args__[0]
```

Comment:
```suggestion
                else typing.get_args(tag.iterable._var_type)[0]
```
better to use the new-style typing functions ala REF-971